### PR TITLE
Fix title behind navbar

### LINKED ISSUE #759: 404 page: Title is behind the navbar
### What happened?

The CSS of the 404 error page is off and adjusts so that you are unable to see the title as it is put behind the navbar. This issue only occurs on certain devices with a different screen size. A smaller screen size will face this issue.

image

### PR DESCRIPTION
**Pull request by nghuyhoang0204**
Hey i've checked this issue and i think the problem is about the margin-top in CSS of this page so i've added and here is the result 
Screenshot 2023-03-15 at 01 04 22

### CODE REVIEW

**Review by k4rni — CHANGES_REQUESTED** [review]
Thank you so much for submitting your code changes! I respect your view of simplicity when it comes to this issue. However, adding a specific value for padding can lead to inconsistent and unpredictable results making it harder to reapproach this page in the future for changes.

Instead, we should focus on centering the content itself. While I appreciate your efforts to improve the layout, I think we need to take a more deliberate approach to ensure that the content is centered correctly.

In order to achieve this, I recommend that you take a look at the HTML and consider putting the content in the `section` tag into a `div` tag so that it will be easy to center
image

From there you should try to see if it is possible to center it without any numeric values for the sake of preventing brute forced programming.

**Review by EthanThatOneKid — COMMENTED** [review]
> Instead, we should focus on centering the content itself.

The suggestion to center the 404 page content in a new DIV is a valid heuristic. 

Note: The DIV might need a height >100vh since a height of 100vh might be too small.

**Comment by nghuyhoang0204** [comment]
Okayy i see what i have to do for this issue . Thanks for your review and detail suggestion
. Ill try to make another Pr 😸

**Comment by EthanThatOneKid** [comment]
> . Ill try to make another Pr 😸

You don’t have to make a new PR, this one is fine to move forward with.

**Comment by nghuyhoang0204** [comment]
Hello guys , so i've made a few changes to this code sorry if my codes are not clean , please check it and give me comment if i have to fix anything else 
And here is the result : 
Screenshot 2023-03-20 at 01 04 07

File: src/lib/components/button/button.svelte
Comment (EthanThatOneKid) [inline src/lib/components/button/button.svelte:1]: The `src/lib/components/button/button.svelte` component isn't within the scope of this PR. Undo any changes in this file.

File: src/routes/+error.svelte [hunk 1]
```diff
@@ -12,16 +12,20 @@
...
+    </div>
+    <img src="/assets/png/lost-frank.png" alt="404 - Page Not Found" />
+  </div>
 </section>
 
 <Footer />
 
 <style>
-  section {
+  /* section {
```
Comment (EthanThatOneKid) [inline src/routes/+error.svelte]: Nit: Delete unused comments.

**Review by EthanThatOneKid — CHANGES_REQUESTED** [review]
> please check it and give me comment if i have to fix anything else

Nice! I left a few comments.

File: src/lib/components/button/button.svelte
Comment (nghuyhoang0204) [reply src/lib/components/button/button.svelte:1]: Hello , if i undo these changes of this file button , the result will be like this 
Screenshot 2023-03-20 at 00 02 39
Do you still accepte

**Comment by k4rni** [comment]
image
I would take just a look again because a change of yours actually affected the landing page!

File: src/lib/components/button/button.svelte
Comment (EthanThatOneKid) [reply src/lib/components/button/button.svelte:1]: > Do you still accepte

I think it is possible to fix the undesired width by putting the button in a container with a set width.

**Comment by nghuyhoang0204** [comment]
Okay guys , i see what i've done so this is my newest commit for yours changes requested . I hope it will be fine now 😄 
Screenshot 2023-03-20 at 01 04 07

File: src/lib/components/button/button.svelte [hunk 2]
```diff
@@ -26,6 +26,7 @@
     user-select: none;
     transition: 0.25s ease-in-out;
 
+
```
Comment (EthanThatOneKid) [inline src/lib/components/button/button.svelte]: Nit: Remove unused linebreak.

File: src/routes/+error.svelte [hunk 3]
```diff
@@ -12,37 +12,34 @@
...
     flex-direction: column;
     justify-content: center;
     align-items: center;
-    margin: 0 24px;
-    color: var(--acm-light);
-  }
-
-  section h1 {
-    align-items: center;
+    height: 120vh;
```
Comment (EthanThatOneKid) [inline src/routes/+error.svelte]: I thought that giving the section a height of >100vh would do the trick. It is close, but not exactly what we need.

What we could do is simply add back your technique from earlier by setting padding to the top of the container to offset the space left under the navbar.

With this offset `padding-top: 100px`, we may remove `height: 120vh` _in favor of the existing `min-height: 100vh`_.

```suggestion
    padding-top: 100px;
```

File: src/routes/+error.svelte [hunk 4]
```diff
@@ -12,37 +12,34 @@
...
 
-  section em {
-    align-items: center;
-    text-align: center;
-    font-size: 50px;
-    text-shadow: 2px 2px #92c4df;
-    font-weight: 600;
+  em {
+    font-size: 40px;
+    text-shadow: rgb(146, 196, 223) 2px 2px;
```
Comment (EthanThatOneKid) [inline src/routes/+error.svelte]: Nit: Use available CSS variable.
```suggestion
    text-shadow: var(--acm-blue) 2px 2px;
```

**Review by EthanThatOneKid — CHANGES_REQUESTED** [review]
Your PR looks good so far. I left a few more comments for you to consider.

**Comment by nghuyhoang0204** [comment]
I am very happy to follow your requests

**Review by EthanThatOneKid — APPROVED** [review]
LGTM! 🎉 

Thank you for another contribution, @nghuyhoang0204!

File: src/routes/+error.svelte (hunk 4 above)
Comment (EthanThatOneKid) [reply src/routes/+error.svelte]: Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/815/commits/af5cc8fd621356f807183cc4db231fe192b041d4>.

File: src/routes/+error.svelte (hunk 3 above)
Comment (EthanThatOneKid) [reply src/routes/+error.svelte]: Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/815/commits/af5cc8fd621356f807183cc4db231fe192b041d4>.

File: src/lib/components/button/button.svelte (hunk 2 above)
Comment (EthanThatOneKid) [reply src/lib/components/button/button.svelte]: Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/815/commits/af5cc8fd621356f807183cc4db231fe192b041d4>.

File: src/routes/+error.svelte (hunk 1 above)
Comment (EthanThatOneKid) [reply src/routes/+error.svelte]: Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/815/commits/f1cb7fb2574a82a88b7e0f7038a0c15013fe9b91>.

File: src/lib/components/button/button.svelte
Comment (EthanThatOneKid) [reply src/lib/components/button/button.svelte:1]: Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/815/commits/f1cb7fb2574a82a88b7e0f7038a0c15013fe9b91>.

**Comment by nghuyhoang0204** [comment]
With my pleasure , thanks for your reviews and your comments 🤓🤩

**Review by k4rni — APPROVED** [review]
LGTM! Great job:)

**Pull request merged**